### ISSUE TITLE
The vehicle cant work well

### ISSUE THREAD

**Issue by Nagisa-233** [Joshua-Ashton/VPhysics-Jolt#48]
The vehicle's wheel cant work, just the hl2 jeep dont have problem ,some vehicle use the simgphys  cant do anything .

**Comment by Benn20002** [comment]
Yeah, pretty much every vehicle I tested (CRSK and TDM Cars) except the HL2 Jeep dont drive. If you press WASD nothing happens except the front wheels turning. Also on my modded DarkRP Cars start bouncing around pretty bad. On default DarkRP you can just walk against vehicle and the move like you're pushing a normal prop or you get killed.

**Comment by gaino1122** [comment]
TDM still dosent work
